diff --git a/lib-cjs/index.js b/lib-cjs/index.js
index b83b6f863b21ccf54556d9cb21877daca7ea32ec..93f85d4bfac69e53925d2b84d2bea6161ed13907 100644
--- a/lib-cjs/index.js
+++ b/lib-cjs/index.js
@@ -24,9 +24,9 @@ var __importStar = (this && this.__importStar) || function (mod) {
 };
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.LibsqlConnection = exports.LibsqlDriver = exports.LibsqlDialect = exports.libsql = void 0;
-const libsql = __importStar(require("@libsql/client"));
+const libsql = __importStar(require("@libsql/client/web"));
 const kysely = __importStar(require("kysely"));
-exports.libsql = __importStar(require("@libsql/client"));
+exports.libsql = __importStar(require("@libsql/client/web"));
 class LibsqlDialect {
     #config;
     constructor(config) {
diff --git a/lib-esm/index.js b/lib-esm/index.js
index 40b13727cef5576c030a8aa70e299cf0956ce550..3e00ad8871d0bafc408a7ba8ad814cee47b8070b 100644
--- a/lib-esm/index.js
+++ b/lib-esm/index.js
@@ -1,6 +1,6 @@
-import * as libsql from "@libsql/client";
+import * as libsql from "@libsql/client/web";
 import * as kysely from "kysely";
-export * as libsql from "@libsql/client";
+export * as libsql from "@libsql/client/web";
 export class LibsqlDialect {
     #config;
     constructor(config) {
